python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py

```diff
@@ -30,13 +30,15 @@ def dask_client():
             yield dask_client
 
 
+@pytest.mark.skip(reason="TODO: Fix these stumpy tests to work with dask")
```

Comment:
I haven't seen these stumpy failures before. Is the monkey patching in this PR causing these failures?

Reply:
Do you mind checking if you haven't? It it is, I'd rather xfail the cugraph tests in this PR (which I think are passing now). And follow-up and add the monkey patching in a way that the integration tests pass.

Reply:
I could be wrong about the monkeypatching causing the stumpy failures.

Reply:
It is not being caused by monkey-patching `from_pandas` in this PR. The stumpy distributed failures seem to be related to some ptx failures. I'm investigating it, but I didn't want them to block this PR. Let's proceed with this PR and then tackle the stumpy failures separately?

Reply:
Yup, thanks for looking into it!